Nav.php

```diff
@@ -202,6 +202,32 @@ public function renderItem($item)
     }
 
     /**
+     * Check if parent item is visible
+     * @param array $parentItem
+     * @return boolean
+     */
+    protected function isParentVisible($parentItem)
```

Comment:
Name sounds like you are checking if item, which is parent to the given one, is chceked to be visible.